src/v/cluster/tests/commands_serialization_test.cc

```diff
@@ -29,13 +29,9 @@
 #include <bits/stdint-intn.h>
```

Comment:
Please keep the commit subject line limited to around 50 characters.

Reply:
github doesn't support comments about commit message, so this is just a comment on the first LOC in the commit but is in refernence to the commit message.

Reply:
`CONTRIBUTING.md` would need updating in that case. I'll re-force-push with shorter commit subject lines.

Reply:
Thanks, I'll get that file updated.